content/posts/chapelcon25-retro/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Reflections on ChapelCon '25"
+date: 2025-11-05
+tags: ["ChapelCon", "Community"]
+series: []
+summary: "This post is a retrospective on ChapelCon '25"
+weight: 15
+authors: ["Brandon Neth"]
+---
+
+ChapelCon '25 is behind us, and another year of productive HPC programming lies ahead!
```

Comment:
Something about "is behind us" sounds a bit like "good riddance" to me.  Maybe "has wrapped up"